### PR TITLE
fixes the sort and name licence fn where no name exists

### PR DESCRIPTION
**Pull request by thurstontye**
If a licence does not have a name property then page will not render because of `license.name.trim()`
 🐿 v2.5.16

### CODE REVIEW

**Review by kateloschinina — APPROVED** [review]

**Comment by kateloschinina** [comment]
LGTM 👍

**Review by leggsimon — APPROVED** [review]

**Comment by leggsimon** [comment]
You'll need to add `@financial-times/n-gage` to the `package.json` to fix that build fyi

File: README.md [hunk 1]
```diff
@@ -33,34 +33,34 @@ If you see an error like `Error: Node Sass does not yet support your current env
 ## Testing
 We are using [Jest](https://facebook.github.io/jest/) for testing React components, and [Enzyme](http://airbnb.io/enzyme/) for rendering components inside our tests.
 
-**Please note that due to using an earlier version of React, we are using version 1.4.x of Enzyme, so not all of the latest functions are available to us. Please refer to [the docs for Enzyme 1.4.x](https://github.com/airbnb/enzyme/tree/442147f669abace1eeae08040885893894ae0505/docs) for a list of available functions.**
+**Please note that due to using an earlier version of React, we are using version1.4.x of Enzyme, so not all of the latest functions are available to us. Please refer to [the docs for Enzyme1.4.x](https://github.com/airbnb/enzyme/tree/442147f669abace1eeae08040885893894ae0505/docs) for a list of available functions.**
```
Comment (leggsimon) [inline README.md:36]: 😬

File: README.md [hunk 2]
```diff
@@ -33,34 +33,34 @@ If you see an error like `Error: Node Sass does not yet support your current env
...
+-   [kmt-myft](https://github.com/Financial-Times/kmt-myft)
+-   [kat-usage-report](https://github.com/Financial-Times/kat-usage-report)
 
- ### How to use the component
+### How to use the component
 
 ## Installation:
 ```
-bower install --S kmt-header
+bower install- -S kmt-header
```
Comment (leggsimon) [inline README.md:63]: 😬

File: README.md [hunk 3]
```diff
@@ -95,50 +95,50 @@ KmtHeader.init(options);
...
+	}
 }
 ```
 
 * Normal standalone use:
 
 ```js
 rootEl: "#root"
-// {String|DOM element} - optional - Query string or DOM element inside which the KMT Header will be placed.
+// {String|DOM element}-   optional-   Query string or DOM element inside which the KMT Header will be placed.
```
Comment (leggsimon) [inline README.md:127]: 😬

**Pull request merged**